components/main-chef-wrapper/main.go

```diff
@@ -34,18 +33,24 @@ func main() {
 	}
 	var (
 		subCommand = os.Args[1]
-		allArgs    = os.Args[2:]
+		allArgs    = os.Args[1:]
 		cmd        *exec.Cmd
 	)
 
 	switch subCommand {
-	// 1) On the creation of new CLIs, for instance, a new CLI called chef-analyze that
```

Comment:
These thoughts and others here has comments have been more fully detailed in https://github.com/chef/chef-workstation/blob/master/docs/Chef-CLI-Catalog.md, so I've removed them from here.